testsuite/integration-arquillian/tests/other/console/src/main/resources/themes/qe/login/messages/messages_en.properties

```diff
@@ -0,0 +1,4 @@
+doAccept=Yes
```

Comment:
OK - agreed. I'll do the following. Hopefully these updates (and please feel free to point out any others) will be enough to merge this PR.

- I'll rename the test to CustomThemeTest and rename the test method to reflect that it tests only the T&C content.
- I'll remove the strings doAccept=Yes, etc.
- I'll also rename the theme to CustomTheme 
- I'll create a new QE KPR for the custom theme test.
- I'll add the test cases into CutomThemeTest in a new PR